src/App.tsx

```diff
@@ -17,11 +17,8 @@ export const App: React.FC<{}> = () => (
             <Route exact path="/">
               <HomePage />
             </Route>
-            {ITProfessions.map(profession => (
-              <Route path={`/details/${profession.route}`} key={profession.id}>
-                <DetailPage profession={profession} />
-              </Route>
-            ))}
+            <Redirect exact path="/details" to={`/details/${ITProfessions[0].route}`} />
+            <Route path="/details/:professionId" component={DetailPage} />
```

Comment:
Hm, that's a good point. However, to avoid a comment I thought of another solution: I put the `to=` path in a constant variable and named it properly (`DefaultDetailsPagePath`). Hope that's better now!